src/TestBundle/Tests/E2E/EditGroupTest.php

```diff
@@ -16,51 +16,51 @@ class EditGroupTest extends WebDriverTestCase
     public function setUp()
     {
         parent::setUp();
-        
+
         $this->logInAsUser('user-1');
         $this->waitForLoadingAnimation();
         $this->webDriver->findElement(
             //First group's edit group button
-            WebDriverBy::cssSelector('ul.groups > li:nth-child(1) > div > a:nth-child(2)')
+            WebDriverBy::cssSelector('.groups .widget-box:nth-child(1) .widget-header .right a:nth-child(1)')
         )->click();
         $this->waitForLoadingAnimation();
     }
-    
+
     public function testDisplaysGroupAndUsers()
     {
-        $this->assertEquals(TestGroupFixtures::$groups['group-1']->getName(), $this->getHeaderText());
+        $this->assertContains(TestGroupFixtures::$groups['group-1']->getName(), $this->getHeaderText());
         $this->assertCount(2, $this->getUsers());
-        $this->assertContains(UserFixtures::$users['user-1']->getEmail(), $this->getUsers()[0]->getText());
-        $this->assertContains(UserFixtures::$users['user-2']->getEmail(), $this->getUsers()[1]->getText());
+        $this->assertContains(strtoupper(UserFixtures::$users['user-1']->getEmail(), $this->getUsers()[0]->getText()));
```

Comment:
@DesignyourCode You've wrapped the strtoupper around both arguments instead of just the first:
```
$this->assertContains(strtoupper(UserFixtures::$users['user-1']->getEmail()), $this->getUsers()[0]->getText());
```